frontend_tests/node_tests/compose.js

```diff
@@ -977,4 +978,24 @@ test_ui("narrow_button_titles", () => {
     );
 });
 
+test_ui("poll_modal", () => {
+    const $poll_options_list = $("#add-poll-form .poll-options-list");
+
+    // simulating adding 1st initial option row
+    const $option_row = $("<opt-stub>");
+    $poll_options_list.set_find_results(".option-row", [$option_row]);
+    const $delete_btn = $("<dtl-btn-stub>");
+    $delete_btn.show();
+    $poll_options_list.set_find_results(".option-row .delete-option", $delete_btn);
+    $option_row.set_find_results(".delete-option", $delete_btn);
+
+    // testing 1st delete button is hidden after 1st initial option row is added
+    poll_modal.update_option_delete_btn($poll_options_list, false);
```

Comment:
Instead of calling this function, we could use the `add_option` and `remove_option` functions, sort of end-to-end testing the module.

Reply:
Not feasible due to limits and unsuitability of zjquery for this case, so current tests will have to do